generateViewModels.js

```diff
@@ -81,5 +81,5 @@ var download = function (url, tsPath, models2get) {
   });
 }
 
-download(newsApiUrl, "/news", ["/dataIndex", "/dataModel", "/post", "/document", "/asset", "/keyValuePair", "/category", "/ministry", "/ministries", "/minister", "/contact", "/resourceLink.ts"]);
-
+//download(newsApiUrl, "/news", ["/dataIndex", "/dataModel", "/post.ts", "/document", "/asset", "/keyValuePair", "/category", "/ministry", "/minister", "/contact", "/resourceLink.ts"]);
```

Comment:
We should think about adding configuration files for the generator instead of using these commented out lines.
eg. npm run gen --configuration=dev.json where dev.json = {
  url: "http://hub.api.dev.com",
  ...
}

also -> how come the post has a .ts and nothing else does?